NeoML/Python/neoml/Dnn/FullyConnected.py

```diff
@@ -39,38 +71,49 @@ def __init__(self, input_layers, element_count, is_zero_free_term=False, name=No
 
     @property
     def element_count(self):
-        """
+        """Gets the length of each vector in the output.
         """
         return self._internal.get_element_count()
 
     @property
     def zero_free_term(self):
-        """
+        """Sets the length of each vector in the output.
         """
         return self._internal.get_zero_free_term()
 
     @zero_free_term.setter
     def zero_free_term(self, zero_free_term):
-        """
+        """Checks if the free term is all zeros.
         """
         self._internal.set_zero_free_term(bool(zero_free_term))
 
 
     def apply_batch_normalization(self, layer):
+        """Connects the batch normalization layer directly 
```

Comment:
_Connects_ wrong word, because when training batch_norm is already _connected_ to this layer's output. Applies/merge/fuses etc.